skyvern/forge/sdk/workflow/service.py

```
@@ -688,6 +693,7 @@ async def _execute_workflow_blocks(
         workflow_run: WorkflowRun,
         organization: Organization,
         browser_session_id: str | None = None,
+        browser_profile_id: str | None = None,
```

Ruff (0.14.3)

Unused method argument: `browser_profile_id`

(ARG002)